sle15/profiles/stig.profile

```diff
@@ -190,9 +189,8 @@ selections:
     - package_aide_installed
     - package_audit-audispd-plugins_installed
     - package_audit_installed
-    - package_SuSEfirewall2_installed
     - package_telnet-server_removed
-    - package_vsftpd_removed
```

Comment:
Not sure why this is showing up as a deleted line. I don't see it in the current sle15/profiles/stig.profile

Reply:
I can see it here: https://github.com/ComplianceAsCode/content/blob/65885c18f89a2696358abf015eb98cb7e25dcff7/sle15/profiles/stig.profile#L195

You probably need to rebase your branch

Reply:
Could be, but this change was started https://github.com/ComplianceAsCode/content/master

Reply:
Fixed, I do see it in my remotes now, not sure why not before. I am religious about remote updates.